cogs/study.py

```diff
@@ -42,6 +54,15 @@ async def on_timeout(self) -> None:
 
         await self.message.edit(view=self)
 
+    # Prevent anyone other than the question asker or helpers from using this
+    async def interaction_check(interaction):
+        helpers = [role.id for role in interaction.channel.changed_roles if "Helper" in role.name]
+
+        interaction_creator_id = interaction.user.id
```

Comment:
defining `interaction_creator_id` is unnecessary here, since it is only accessed once. Perhaps we could call `interaction.user.id` directly in the `return` statement?